DataManagementSystem/Service/FTS3ManagerHandler.py

```diff
@@ -77,7 +77,7 @@ def export_getOperation( cls, operationID ):
     return S_OK( opJSON )
 
 
-  types_getActiveJobs = [ ( long, int ), [None ] + list( basestring ), basestring ]
+  types_getActiveJobs = [ ( long, int ), [None ] + [ basestring ], basestring ]
```

Comment:
hum, search/replaced too quickly I guess. But I think the correct fix here is to put 
``` basestring``` and not ```[basestring]```.  (though it probably does not harm ! )